File: JAVA/PROGRAM/Harddrive.java
public class Harddrive extends Komponen {
    private int kapasitasGb;
    private String tipeDrive;
    
    // Konstruktor
    public Harddrive() {
        super();
        this.kapasitasGb = 0;
        this.tipeDrive = "";
    }
    
    public Harddrive(int kapasitasGb, String tipeDrive, String merk, String nama) {
        super(merk, nama);
        this.kapasitasGb = kapasitasGb;
        this.tipeDrive = tipeDrive;
    }
    
    // Getter dan Setter
    public int getKapasitasGb() {
        return kapasitasGb;
    }
    
    public void setKapasitasGb(int kapasitasGb) {
        this.kapasitasGb = kapasitasGb;
    }
    
    public String getTipeDrive() {
        return tipeDrive;
    }
    
    public void setTipeDrive(String tipeDrive) {
        this.tipeDrive = tipeDrive;
    }
}

File: PYTHON/PROGRAM/Harddrive.py
from Komponen import Komponen

class Harddrive(Komponen):
    def __init__(self, kapasitas_gb=0, tipe_drive="", merk="", nama=""):
        super().__init__(merk, nama)
        self.__kapasitas_gb = kapasitas_gb
        self.__tipe_drive = tipe_drive
    
    # Getter
    def get_kapasitas_gb(self):
        return self.__kapasitas_gb
    
    def get_tipe_drive(self):
        return self.__tipe_drive
    
    # Setter
    def set_kapasitas_gb(self, kapasitas_gb):
        self.__kapasitas_gb = kapasitas_gb
    
    def set_tipe_drive(self, tipe_drive):
        self.__tipe_drive = tipe_drive